### DISCUSSION TITLE
وبسایت های فیلم و سریال اشتراکی - دارای حذفیات

### DISCUSSION THREAD

**Discussion by neganmj** [MasterKia/PersianBlocker#230] [Ideas]
سلام.
من تازه با پروژه شما آشنا شدم و واقعا لذت بردم، حرف ندارید، دیتابیس پروژه رو بررسی کردم، یسری آدرس های سایت های فیلم و سریال قدیمیه و بروز نیست و جای خالی سایتای بیشتری هم حس میشه پس اومدم براتون قرارشون بدم، یسری سایت ها هستن که به صورت اشتراکی کار میکنن برای ارائه خدمات دانلود و تماشای فیلم و سریال هاشون
اینجا تو این صفحه من براتون آدرس هاشونو به مرور زمان مینویسم تا به لیستتون اضافه کنید.

https://oxymovie.com/ ( اشتراکی )
https://animelist.tv/ (اشتراکی)
https://playrooz.com/ (اشتراکی)
https://www.dlday.click/ (اشتراکی)

زیر مجموعه سایت upera:
https://persianvid.com/ ( دارای حذفیات )
https://clickfiles.ir/ ( دارای حذفیات )

زیر مجموعه ساترا :
https://www.filmdoon.com/ ( دارای حذفیات )
https://hexdl.com/ ( دارای حذفیات )

**Comment by MasterKia** [comment]
درود بر شما، ممنون از محبتتون.
به لیست افزوده شدن. شرمنده بابت تأخیر.

بین اون سایت‌های اشتراکی، احیاناً هیچ‌کدومشون محتوای رایگان ندارن که؟

**Reply by neganmj** [reply]
ممنون لطف دارید

سایت اول، سوم و چهارم طبق بررسی سطحی که کردم، هم سریال و هم فیلم هاشون نیاز به تهیه اشتراک داره ( بازم خودتون یه نیم نگاه بندازید بهتره )
سایت دوم که انیمه میزاره، انیمه سریالیشو چک نکردم ، ولی سینمایی هاش بدون اشتراک با کیفیت 480 فقط قرار میده، برای کیفیت بهتر باید اشتراک تهیه بشه.
